akka-docs/src/main/paradox/cluster-sharding.md

```diff
@@ -401,18 +427,13 @@ you can disable durable storage and benefit from better performance by using the
 akka.cluster.sharding.distributed-data.durable.keys = []
 ```
 
-When `rememberEntities` is set to false, a `Shard` will not automatically restart any entities
-after a rebalance or recovering from a crash. Entities will only be started once the first message
+### Behavior When Not Enabled 
+
+When `rememberEntities` is disabled (the default), a `Shard` will not automatically restart any entities
+after a rebalance or recovering from a crash. Instead, entities are started once the first message
 for that entity has been received in the `Shard`. Entities will not be restarted if they stop without
 using a `Passivate`.
```

Comment:
That last sentence is probably just adding confusion and can be removed. It will be started again when next message arrives, also if it is stopped without Passivate.